### PR TITLE
chore(jest): defaultbrowsercontext.jest.js

### PR DESCRIPTION
**Pull request by JoelEinbinder**

### CODE REVIEW

File: test/defaultbrowsercontext.jest.js [hunk 1]
```diff
@@ -18,25 +18,32 @@
...
-    return state;
-  }
-  async function close(state) {
-    await state.context.close();
-    await removeUserDataDir(state.userDataDir);
-    delete state.page;
-    delete state.context;
-    delete state.userDataDir;
-  }
+  let userDataDir;
```
Comment (pavelfeldman) [inline test/defaultbrowsercontext.jest.js:24]:
```js
registerFixture('launch', async ({browserType, defaultBrowserOptions}) => {
    const userDataDir = await makeUserDataDir();
    const context = await browserType.launchPersistentContext(userDataDir, {...defaultBrowserOptions, ...options});
    await test({ userDataDir, context, page: context.pages()[0] });
    await context.close();
    await removeUserDataDir(userDataDir);
});
```

Comment (pavelfeldman) [reply test/defaultbrowsercontext.jest.js:24]: I can see your launch is a function with parameters, you can have it being a function too!

Comment (JoelEinbinder) [reply test/defaultbrowsercontext.jest.js]: made it a fixture

File: test/jest/fixtures.js [hunk 2]
```diff
@@ -153,6 +157,27 @@ module.exports = function registerFixtures(global) {
...
+  global.registerFixture('launchPersistent', async ({userDataDir, defaultBrowserOptions, browserType}, test) => {
+    let context;
+    async function launchPersistent(options) {
+      if (context)
+        throw new Error('can only launch one persitent context');
+      context = await browserType.launchPersistentContext(userDataDir, {...defaultBrowserOptions, ...options});
+      const page = context.pages()[0];
+      return {context, page};
+    }
+    await test(launchPersistent);
```
Comment (pavelfeldman) [inline test/jest/fixtures.js:175]: try/catch

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**